README.md

```diff
@@ -79,6 +79,17 @@ with:
   githubToken: ${{ secrets.GITHUB_TOKEN }}
   caCertificate: ${{ secrets.VAULTCA }}
 ```
+- **jwt**: you must provide a `role` & `privateKey` parameters, additionally you can pass `keyPassword` & `jwtTtlSec` parameters
+```yaml
+...
+with:
+  url: https://vault.mycompany.com:8200
+  method: jwt
+  role: github-action
+  privateKey: ${{ secrets.JWT_PRIVATE_KEY }}
```

Comment:
Other [multiline parameters](https://github.com/hashicorp/vault-action#reference) in the project are base64 encoded. I think it makes sense to be consistent here with this private key and decode it when used. Thoughts?

Reply:
Thank you for pointing out, I didn't notice that. I have fixed it now.